ui/admin-portal/src/app/components/opportunity/edit-opp/edit-opp.component.ts

```diff
@@ -62,7 +80,42 @@ export class EditOppComponent implements OnInit {
       nextStep: this.nextStep,
       nextStepDueDate: this.nextStepDueDate,
     }
-    this.activeModal.close(info)
+    const parentJobInfo: OpportunityProgressParams = {
+      nextStep: this.nextStep,
+      nextStepDueDate: this.nextStepDueDate,
+    }
+    if (this.copyToParentJob && isCandidateOpportunity(this.opp)) {
+      this.jobService.update(this.opp.jobOpp.id, parentJobInfo).subscribe(() => {
+          this.activeModal.close(info);
+        },
+        (error) => this.error = error);
+      } else {
+        this.activeModal.close(info)
+    }
   }
 
+  /**
+   * If opp is the only open case of its parent job, sets isOnlyOpenCaseOfParentJob to true — the job has a ShortSavedList property but checking for open cases requires the full SavedList, so this is retrieved before proceeding.
+   * @param opp CandidateOpportunity/case whose progress is being updated
+   *
+   */
+  isOnlyOpenCaseOfParentJobCheck(opp: CandidateOpportunity) {
```

Comment:
Sorry to be a pain, but it would be good to get this right.

If you have a method starting with "is" people will naturally assume that the method will return true or false.

So this is confusing. A better name for the method would be "checkOpenCases" or something like that.